.travis.yml

```diff
@@ -3,8 +3,9 @@ language: go
 go:
   - "1.13"
   - "1.14"
+  - "1.17"
```

Comment:
what's the strategy here? It prefer higher version?

Reply:
I think it was used to test various go env.
We will remove Travis in this PR: https://github.com/kubeflow/testing/pull/967, so I can remove this change.